README.md

```diff
@@ -137,7 +138,7 @@ git commit -m 'change the url name.'
 
 
 
-#### Publish to Your gh-pages Branch
+#### Publish Your gh-pages Branch to Your cesehub Folk on GitHub)
 
```

Comment:
Ops, sorry... I understand the problem now.
1. that right parenthesis should be removed...
2. typo "termial" is in other paragraph.
3. it should be fork as I originally wanted to say.

But I didn't really know if my expression is correct or not. I wanted to emphasis that this publishing was to publish your gh-pages to user's private git repository. 
If you think I use the wrong/inaccurate way to express this thing, please let me know. I am happy to learn the right expression of this.